CHANGES/5741.bugfix

```diff
@@ -0,0 +1,3 @@
+Fixes missing comps group attrs when publishing centos7 remote
```

Comment:
I believe it's not specific to centos7, it's a fix for any comps that contain this data.
How about  "Fix comps.xml publish: missing group attributes desc_by_lang, name_by_lang, and default now appear properly." ?  Or something along those lines and if we can keep it as a one line, it would be perfect.